internal/server/http/test/testdata/imposters_variables/responses/gopher_1_2_response.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can we have different attributes (name, color and age) for each, please? I know it may sound nitpicky, but when things go wrong, this is the kind of things that makes the difference at time of finding the issue.